crates/lsp/src/config.rs

```diff
@@ -101,7 +101,7 @@ impl From<VscDocumentConfig> for DocumentConfig {
                 if var == "tabSize" {
                     x.tab_size
                 } else {
-                    crate::log_warn!("Unknown indent alias {var}, using default");
+                    log::warn!("Unknown indent alias {var}, using default");
```

Comment:
While its possible to move `auxiliary_events_tx` into here, it didn't feel worth it for this one? Happy to do it though.